config/locales/en.yml

```diff
@@ -19,7 +19,7 @@ en:
       approve_sub_heading: Get the changes approved and merged into master.
     index:
       landing_page_edit:
-        accordions: Edit landing page accordions and announcements
+        accordions: Edit landing page accordions, announcements and timeline entries
```

Comment:
This is getting a bit long now! I wonder if instead, the `something_else` link should be changed to "Publish changes from GitHub" and the `accordions` link could be shortened to "Edit landing page"?

Reply:
@leenagupte yeah it is long! I'll ask some of the content designers tomorrow to see what they prefer :)